app/src/main/res/layout/activity_main.xml

```diff
@@ -1,32 +1,28 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
+<?xml version="1.0" encoding="utf-8"?><!--
   ~ Copyright (c) 2020 Plaid Technologies, Inc. <support@plaid.com>
   -->
 
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
-  xmlns:app="http://schemas.android.com/apk/res-auto"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Wondering: is a `LinearLayout` with weights easier/harder to understand than a `ConstraintLayout`?

Reply:
I find `ConstraintLayout` 1. difficult to read and 2. requires an extra library. LL with weights can be confusing with what the weight will look like, but at least you're guaranteed that the views will show up in the order that is written (vs ConstraintLayout's   `app:layout_constraintStart_toStartOf="@+id/randomThingInTheMiddleOfTheScreen"`)